### PR TITLE
Rollback third_party/dart-sdk to 1.13.0

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

**Comment by eseidelGoogle** [comment]
I presume we let the dart team know. :)  @johnmccutchan

**Comment by jason-simmons** [comment]
Updated the Travis analyzer script

**Comment by eseidelGoogle** [comment]
Thanks @pq to see our continuing pile of analyzer greps. :D

**Comment by johnmccutchan** [comment]
What is the context for the roll back? Anything we need to look into?

**Comment by jason-simmons** [comment]
The "pub_build_observatory" step was taking an unusually long time during the Flutter build

We had also tried moving Flutter to a new version of the Dart VM.  We rolled that back due to an error seen while initializing the service isolate:
    Dart_LoadScript: A script has already been loaded from 'file:///src/flutter/engine/src/sky/engine/bindings/snapshot.dart'

**Comment by eseidelGoogle** [comment]
Do we have a bug on file with the Dart team or someone owning fixing 1.14 to let us roll forward?  @iposva-google

**Comment by johnmccutchan** [comment]
@eseidelGoogle The ```Dart_LoadScript: A script has already been loaded from``` error should be fixed already.

**Comment by eseidelGoogle** [comment]
I spoke with @johnmccutchan in person, he noted that pub_build_observatory *can* take a long time, and that that is unlikely to have changed between our previous sdk (the binary files dart, dartanalyzer, etc.) and 1.1.4.  But generally you only pub_build_observatory (convert from dart to js) the first time you build and then never again since it doesn't change.  It would have rebuilt due to our dart2js binary changing of course.

TL;DR: We should try rolling again.  @jason-simmons are you the right person to do that?

**Comment by johnmccutchan** [comment]
@jason-simmons Ping me if you need assistance.

**Comment by jason-simmons** [comment]
Just tried a roll to 1.14.1.  Flutter is working as expected, but pub_build_observatory is still taking a full minute during each build of the Flutter engine.

**Comment by johnmccutchan** [comment]
I've fixed the rebuilds in 3bf51e01386a0b0d12dc6655a69b58e3f67fcf02

**Comment by johnmccutchan** [comment]
CL with some explanation of what was going on: https://codereview.chromium.org/1664403002/
